src/content/app/tools/blast/state/action-listeners/blastActionListeners.test.ts

```diff
@@ -0,0 +1,243 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { configureStore, clearAllListeners } from '@reduxjs/toolkit';
+import { waitFor } from '@testing-library/react';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+
+import createRootReducer from 'src/root/rootReducer';
+import listenerMiddleware, {
+  startAppListening
+} from 'src/root/actionListenerMiddleware';
+
+import {
+  submitBlastListener,
+  resforeBlastSubmissionsListener
+} from 'src/content/app/tools/blast/state/action-listeners/blastActionListeners';
+
+import restApiSlice from 'src/shared/state/api-slices/restSlice';
```

Comment:
Move this import below to the group of state imports?

Reply:
✅